jobbergate-slurp/slurp/db_login.py

```diff
@@ -0,0 +1,5 @@
+def login(is_legacy=typer.):
```

Comment:
Is this a typo?

Reply:
yes, it sure is. Not sure how that got in there.